search-resources-api/src/main/java/no/unit/nva/search/SearchResourcesApiHandler.java

```diff
@@ -0,0 +1,54 @@
+package no.unit.nva.search;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.search.exception.InputException;
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.handlers.ApiGatewayHandler;
+import nva.commons.handlers.RequestInfo;
+import nva.commons.handlers.RestRequestHandler;
+import nva.commons.utils.Environment;
+import org.slf4j.LoggerFactory;
+
+public class SearchResourcesApiHandler extends ApiGatewayHandler<SearchResourcesRequest, SearchResourcesResponse> {
```

Comment:
Terminology question, what is a Resource? Is it a Supertype for Publication?

Reply:
Something between... As for now it is the name of the dynamoDB table - but i could comain lots of resources...